package.json

```diff
@@ -165,5 +165,8 @@
   "workspaces": [
     "packages/*",
     "docs"
-  ]
+  ],
+  "dependencies": {
+    "cpy-cli": "^2.0.0"
```

Comment:
I think that the dependency should be added to the `/docs/package.json`.

Reply:
My thinking was rimraf which does the same thing (substituting command) is added in /package.json
But /docs/package.json makes sence too, since it is only used there.